spec.bs

```diff
@@ -2239,3 +2239,25 @@ achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-geolocation.https.html
   /fenced-frame/permission-notification.https.html
 </wpt>
+
+<h3 id=cssom-monkeypatch>CSSOM View</h3>
```

Comment:
In this section please describe why we cannot allow this behavior (aka what privacy concern we're avoiding by implementing this section of the spec)